gradle.properties

```diff
@@ -1,7 +1,7 @@
 org.gradle.jvmargs=-Xmx4096m
 android.useAndroidX=true
-android.enableJetifier=true
+android.enableJetifier=false
```

Comment:
I think we should remove this line altogether

Reply:
On second though, we should keep it to true as per https://stackoverflow.com/a/56490827/14790459

Reply:
yeah seems too risky I guess but the fast build times are awesome too. I'm deeply divided xD. Will revert it in the next PR #209.

Reply:
Cool